superflore/generators/ebuild/overlay_instance.py

```diff
@@ -30,30 +30,19 @@ def __init__(self, repo_dir, do_clone, org='ros', repo='ros-overlay'):
         info('Creating new branch {0}...'.format(self.branch_name))
         self.repo.create_branch(self.branch_name)
 
-    def clean_ros_ebuild_dirs(self, distro=None):
-        if distro:
-            info('Cleaning up ros-{0} directory...'.format(distro))
-            self.repo.git.rm('-rf', 'ros-{0}'.format(distro))
-        else:
-            info('Cleaning up ros-* directories...')
-            self.repo.git.rm('-rf', 'ros-*')
-
     def commit_changes(self, distro):
         info('Adding changes...')
-        if distro:
-            self.repo.git.add('ros-{0}'.format(distro))
-        else:
-            self.repo.git.add('ros-*')
-            distro = 'update'
-        commit_msg = {
-            'update': 'rosdistro sync, {0}',
-            'all': 'regenerate all distros, {0}',
-            'lunar': 'regenerate ros-lunar, {0}',
-            'indigo': 'regenerate ros-indigo, {0}',
-            'kinetic': 'regenerate ros-kinetic, {0}',
-        }[distro].format(time.ctime())
+        self.repo.git.add(self.repo.repo_dir)
         info('Committing to branch {0}...'.format(self.branch_name))
-        self.repo.git.commit(m='{0}'.format(commit_msg))
+        self.repo.git.commit(m='{0}'.format(
+            {
+                'update': 'rosdistro sync, {0}',
+                'all': 'regenerate all distros, {0}',
+                'lunar': 'regenerate ros-lunar, {0}',
+                'indigo': 'regenerate ros-indigo, {0}',
+                'kinetic': 'regenerate ros-kinetic, {0}',
+            }[distro or 'update'].format(time.ctime()))
```

Comment:
Also this formatting is very hard to read. I'd suggest not doing formatting inside of formatting. A temporary variable or function would make it much easier to understand. Especially in python you want to focus on readability not just compactness.